types/services.d.ts

```diff
@@ -92,7 +92,7 @@ export interface IServiceCommon {
     fieldName: string,
     value: any
   ): Promise<boolean>;
-  sanitizeCommentEntity(entity: Comment): Comment;
+  sanitizeCommentEntity(entity: Comment, populate?: StringMap<boolean | Array<string> | StringMap<unknown>>): Comment;
```

Comment:
```suggestion
  sanitizeCommentEntity(entity: Comment, populate?: PopulateClause): Comment;
```

Once we mark `PopulateClause` as `export`ed